week4/assignment/src/context/SignUpContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useState, useContext, ReactNode } from 'react';
```

Comment:
```suggestion
import { createContext, useState, useContext, ReactNode } from 'react';
```

사용하지 않는 import는 지워주기!